packages/icon/src/Icon.spec.tsx

```diff
@@ -5,6 +5,7 @@ import { toJson } from 'xml2json';
 import { render, cleanup } from '@testing-library/react';
```

Comment:
No test for when title is null?

Reply:
So since we can't actually test the SVGR output, we aren't able to test that the correct title is rendered.

I think there's a decent way we can still do this – I can move all the title logic to a self-contained function we can independently test. I'll re-request when that's done.

Reply:
Alright, let me know what you think of my solution for testing this.